Wrong IP in PingScheduledTask

## Description

I was trying to run the `appsmith-server` module locally but found that after running I get this exception in the terminal:
```
- An exception was thrown by reactor.netty.resources.PooledConnectionProvider$PooledConnectionAllocator$PooledConnectionInitializer.operationComplete()
reactor.core.Exceptions$ErrorCallbackNotImplemented: java.net.UnknownHostException: The requested name is valid, but no data of the requested type was found (api6.ipify.org)
Caused by: java.net.UnknownHostException: The requested name is valid, but no data of the requested type was found (api6.ipify.org)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
Error has been observed at the following site(s):
	|_ checkpoint ⇢ Request to GET https://api6.ipify.org [DefaultWebClient]
Stack trace:
		at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
		at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:928)
		at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1514)
		at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:847)
		at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1504)
		at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1363)
		at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1297)
		at java.base/java.net.InetAddress.getByName(InetAddress.java:1247)
		at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:148)
		at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:145)
		at java.base/java.security.AccessController.doPrivileged(Native Method)
		at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:145)
		at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:43)
		at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:63)
		at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:55)
		at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:57)
		at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:32)
		at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:108)
		at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:200)
		at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
		at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:180)
		at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:166)
		at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:577)
		at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:551)
		at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:490)
		at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:615)
		at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:604)
		at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104)
		at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
		at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:984)
		at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:504)
		at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:417)
		at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:474)
		at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
		at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
		at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
		at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
		at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
		at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
		at java.base/java.lang.Thread.run(Thread.java:834)

```

After further digging, I think that the value of the URL for the below variable could be wrong:
https://github.com/appsmithorg/appsmith/blob/c1b498f6b81dd67d707fd45c7dd40d6990486438/app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/PingScheduledTask.java#L29

I was just trying to explore the code-base so this bug could be of no significance as I am lacking a huge context here. I think the URL should be one of the ones mentioned at https://www.ipify.org/.

### Steps to reproduce the behaviour:
1. Ensure that MongoDB and Redis are running locally and their values are updated in the .env file.
2. Run the appsmith-server module using `ServerApplication.main` method.

### Important Details
 - Version: Local run from the release branch
 - OS: Windows 10 Home

--- a/app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/PingScheduledTask.java
+++ b/app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/PingScheduledTask.java
@@ -26,7 +26,7 @@ public class PingScheduledTask {
 
     private final ConfigService configService;
 
-    public static final URI GET_IP_URI = URI.create("https://api6.ipify.org");
+    public static final URI GET_IP_URI = URI.create("https://api64.ipify.org");
 
     /**
      * Gets the external IP address of this server and pings a data point to indicate that this server instance is live.

